api/lib/handlers/login/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var config = require('../../config.js').config;
+var pg = require('pg');
+
+module.exports.post = function(req, res, next) {
+  var id = req.body.id;
+  var pool = new pg.Pool(config);
+  pool.connect(function(err, client, done) {
```

Comment:
`pg-promise` (that relies on this library) offers promises
`pg-async` for async-await